scripts/build_llvm.sh

```diff
@@ -24,7 +24,7 @@
 LLVM_INSTALL_PREFIX=${LLVM_INSTALL_PREFIX:-$HOME/.llvm}
 LLVM_PROJECTS=${LLVM_PROJECTS:-'clang;lld;mlir;python-bindings'}
 PYBIND11_INSTALL_PREFIX=${PYBIND11_INSTALL_PREFIX:-/usr/local/pybind11}
-Python3_EXECUTABLE=${Python3_EXECUTABLE:-python3}
+Python3_EXECUTABLE=${Python3_EXECUTABLE:-python3.8}
```

Comment:
3.8 is rare to be installed on any machine. Hence please keep it as the more general command. The only place where it should matter which python version is used is the build of the wheel itself, where this variable is already explicitly set. However, clearly there is some bug so if we need to nail down the python version for building the llvm parts that are - or rather should be! - *unrelated to python*, then we should do that be instead setting this environment variable in the devdeps.manylinux.Dockerfile.